modules/otel-collector-components.adoc

```diff
@@ -557,7 +557,39 @@ config: |
 <2> The default exporter when the attribute value is not present in the table in the next section.
 <3> The table that defines which values are to be routed to which exporters.
 
-You can optionally create an `attribute_source` configuratiion, which defines where to look for the attribute in `from_attribute`. The allowed value is `context` to search the context, which includes the HTTP headers, or `resource` to search the resource attributes.
+You can optionally create an `attribute_source` configuration, which defines where to look for the attribute in `from_attribute`. The allowed value is `context` to search the context, which includes the HTTP headers, or `resource` to search the resource attributes.
+
+[id="cumulativetodelta-processor_{context}"]
+=== Cumulative to Delta Processor
+
+:FeatureName: The Cumulative to Delta Processor
+include::snippets/technology-preview.adoc[]
+
+This processor converts monotonic, cumulative-sum, and histogram metrics to monotonic delta metrics.
+
+[NOTE]
+====
+This processor does not convert non-monotonic sums and exponential histograms.
+====
+
+.Example of an OpenTelemetry Collector custom resource with an enabled Cumulative to Delta Processor
+[source,yaml]
+----
+config: |
+  processors:
+    cumulativetodelta:
+      include: # <1>
+        match_type: strict
+        metrics:
+        - first_metric
+        - second_metric
+      exclude: # <2>
+        match_type: regexp
+        metrics:
+        - "metric_.*"
```

Comment:
```suggestion
        - "<metric_.*>"
```

Reply:
Thank you for this suggestion. I checked with the QE and found it to be correct as it is.